connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -381,7 +381,7 @@ message Sample {
 
   // (Optional) Explicitly sort the underlying plan to make the ordering deterministic.
   // This flag is only used to randomly splits DataFrame with the provided weights.
-  optional bool force_stable_sort = 6;
+  optional bool deterministic_order = 6;
```

Comment:
can we update the field doc? it's not always sorting the input plan.

Reply:
And we can mark this field as required.

Reply:
OK